scripts/kubernetes/start_redis.sh

```diff
@@ -24,9 +24,9 @@ title "Starting Rafiki's Redis..."
 LOG_FILE_PATH=$PWD/logs/start_redis_service.log
 (kubectl create -f scripts/kubernetes/start_redis_service.json \
 &> $LOG_FILE_PATH) &
-ensure_stable "Rafiki's Redis Service" $LOG_FILE_PATH 10
+ensure_stable "Rafiki's Redis Service" $LOG_FILE_PATH 5
```

Comment:
what does it mean by changing the number from 10 to 5?

Reply:
wait for 5 sec is enough, we don't need to wait for that longer,